.github/workflows/go.yml

```diff
@@ -34,14 +34,33 @@ jobs:
         with:
           version: "v0.7.0"
 
-      - name: Run Make
-        run: make
+      # TODO(zzxwill) This step and the follow one need to be removed after rudr admin:init is ready
```

Comment:
now it's ready， try using that

Reply:
I verified `vela system:init`, and found it just installed oam runtime, but didn't install definition manifests. Is it by-design?

Reply:
no, we should fix it